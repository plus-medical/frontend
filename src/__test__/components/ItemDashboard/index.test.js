/* eslint-disable no-undef */
import React from 'react'
import { create } from 'react-test-renderer'
import RouterMock from '../../../__mocks__/RoutesMocks'
import ItemDashboard from '../../../components/itemDashboard/index'

describe('<ItemDashboard />', () => {
  test('Review Snapshot in ItemDashboard component', () => {
    const photo = create(
      <RouterMock>
        <ItemDashboard />
      </RouterMock>
    )
    expect(photo.toJSON()).toMatchSnapshot()
  })
})
