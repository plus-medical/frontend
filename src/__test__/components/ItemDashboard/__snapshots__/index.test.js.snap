// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemDashboard /> Review Snapshot in ItemDashboard component 1`] = `
<a
  href=""
  onClick={[Function]}
>
  <div
    className="dashboard-item"
  >
    <div
      className="dashboard-item__img"
    />
    <div
      className="dashboard-item__txt"
    />
    <div
      className="dashboard-item__togo"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 384 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M365.52 209.85L59.22 67.01c-16.06-7.49-35.15-.54-42.64 15.52L3.01 111.61c-7.49 16.06-.54 35.15 15.52 42.64L236.96 256.1 18.49 357.99C2.47 365.46-4.46 384.5 3.01 400.52l13.52 29C24 445.54 43.04 452.47 59.06 445l306.47-142.91a32.003 32.003 0 0 0 18.48-29v-34.23c-.01-12.45-7.21-23.76-18.49-29.01z"
        />
      </svg>
    </div>
  </div>
</a>
`;
